src/seedu/addressbook/Main.java

```diff
@@ -13,117 +13,136 @@
 import java.util.List;
 import java.util.Optional;
 
-
 /**
- * Entry point of the Address Book application.
- * Initializes the application and starts the interaction with the user.
+ * Entry point of the Address Book application. Initializes the application and
+ * starts the interaction with the user.
  */
 public class Main {
 
-    /** Version info of the program. */
```

Comment:
Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes.